bluemix/configuration/core_config/bx_config.go

```diff
@@ -297,10 +297,11 @@ func (c *bxConfig) IAMID() (guid string) {
 }
 
 func (c *bxConfig) IsLoggedIn() (loggedIn bool) {
```

Comment:
```suggestion
func (c *bxConfig) IsLoggedIn() (bool) {
```
we are no longer using `loggedIn`

Reply:
Will do